docs/docs/guides/integrate/identity-providers/migrate.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Migrate from Generic Provider to specific Identity Provider
+sidebar_label: Migrate IDP
+---
+
+## Migrate Generic OIDC Provider
+
+You can migrate from a generic OIDC provider to the following supported templates:
+- AzureAD
+- Google
+
+To migrate you only have to use the [Migrate Generic OIDC Identity Provider (Instance)](/docs/apis/resources/admin/admin-service-migrate-generic-oidc-provider#migrate-generic-oidc-identity-provider) or [Migrate Generic OIDC Identity Provider (Organization)](/docs/apis/resources/mgmt/management-service-migrate-generic-oidc-provider#migrate-generic-oidc-identity-provider) call,
+which only changes the type of the provider and doesn't delete the already linked user to the provider.
+
+### Google Configuration
+
+The available configuration is described in [Google Configuration](./google).
+
+### AzureAD Configuration
+
+The available configuration is described in [AzureAD Configuration](./azure-ad).
+
+## Migrate with Terraform
+
+To migrate to a specific provider there are only a couple steps:
+- Have an already existing IDP, which has to be migrated as to not lose the existing links of users to users of the external IDP
```

Comment:
This sentence sounds a little strange. What do you want to tell? That this is only possible if you have an existing provider? Otherwise you can just create a new one?

I think we can delete it here, I have added a note above